thehive.json

```diff
@@ -3210,42 +3210,285 @@
                 "type": "table"
             },
             "versions": "EQ(*)"
-        }
-    ],
-    "pip_dependencies": {
```

Comment:
Any specific reason to remove this dependency from json?

Reply:
Removed during migration to this repository. Readding